libraries/botbuilder-dialogs/package.json

```diff
@@ -32,10 +32,12 @@
     "@microsoft/recognizers-text-number": "1.1.4",
     "@microsoft/recognizers-text-suite": "1.1.4",
     "botbuilder-core": "4.1.6",
+    "botbuilder-runtime-core": "4.1.6",
```

Comment:
Unfortunately this needs to be shimmed as well

Reply:
It already is